### ISSUE TITLE
Modify Existing Android Apps

### ISSUE THREAD

**Issue by Sid200026** [Sid200026/studious-train#3]
Go through the current repository of apps and then add new features or modify the old ones or remove some bugs.

Raise a new issue with the App name you want to modify and then the feature you added, modified or fixed.

You can look at the README.md for some suggested changes

`[Modification] App_Name : Feature_Changed`

**Comment by javadVatan** [comment]
Could you explain more about the features ?

**Comment by Sid200026** [comment]
I have updated README.md with project descriptions and some suggested changes.

**Comment by Honey14** [comment]
Hi, Can i work on the Timer app ?

**Comment by Sid200026** [comment]
Yes....just raise a new issue and I will assign the issue to you.

**Comment by Honey14** [comment]
I've created.

**Comment by Altamira07** [comment]
Hi, Can I work on the UI for Tic Tac Toe game?

**Comment by Sid200026** [comment]
Just create a new issue for this.. and I will assign it to you.
